src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -543,14 +544,14 @@ private ValueTask<FormulaValue> VisitBinaryOpNode(BinaryOpNode node, EvalVisitor
                 case BinaryOpKind.GeqTime:
                     return OperatorGeqTime(this, context, node.IRContext, args);
                 case BinaryOpKind.DynamicGetField:
-                    return new ValueTask<FormulaValue>(OperatorDynamicGetField(node, args));
+                    return new ValueTask<FormulaValue>(OperatorDynamicGetField(node, args, CultureInfo));
 
                 default:
                     return new ValueTask<FormulaValue>(CommonErrors.UnreachableCodeError(node.IRContext));
             }
         }
 
-        private static FormulaValue OperatorDynamicGetField(BinaryOpNode node, FormulaValue[] args)
+        private static FormulaValue OperatorDynamicGetField(BinaryOpNode node, FormulaValue[] args, CultureInfo locale)
```

Comment:
>, CultureInfo locale [](http://example.com/codeflow?start=98&length=20)

With the new resource localization model, there's no need to pass this argument anymore